### PR TITLE
samples: regional topics samples

### PR DESCRIPTION
**Pull request by anguillanneuf**
DO NOT SUBMIT until service goes live (early 2022). 
DO NOT SUBMIT until we cut a new release of the library that includes #274.

All tests pass using the whitelisted project for now.

Also see https://github.com/googleapis/java-pubsublite/pull/929

### CODE REVIEW

File: samples/snippets/update_lite_topic_example.py [hunk 1]
```diff
@@ -35,39 +40,44 @@ def update_lite_topic(project_number, cloud_region, zone_id, topic_id):
     # cloud_region = "us-central1"
     # zone_id = "a"
     # topic_id = "your-topic-id"
+    # reservation_id = "your-reservation-id"
 
     cloud_region = CloudRegion(cloud_region)
     location = CloudZone(cloud_region, zone_id)
     topic_path = TopicPath(project_number, location, topic_id)
```
Comment (jiangmichaellll) [inline samples/snippets/update_lite_topic_example.py:47]: those all need to be changed so that location can be either zonal or regional

Comment (anguillanneuf) [reply samples/snippets/update_lite_topic_example.py:56]: Updated.

**Comment by jiangmichaellll** [comment]
Do you know why all the "Samples - Python 3.X" tests fail? Can you resolve those, after that ping me and I will LGTM

**Review by jiangmichaellll — APPROVED** [review]

**Comment by parthea** [comment]
Hi @anguillanneuf, please could you take a look?

**Pull request merged**